setup.py

```diff
@@ -44,6 +44,7 @@
     include_package_data=True,
     install_requires=[
         "pytket ~= 1.15",
+        "pytket-qir == 0.2.0rc1",
```

Comment:
Do we want to have this as a dependency or do we want to do something like ask the user to install it if they want to use this feature? I would assume that this could generated problems, because pytket-qir is not supporting arm64 linux?